WordPress/src/main/java/org/wordpress/android/ui/reader/repository/FetchPostsForTagUseCase.kt

```diff
@@ -0,0 +1,21 @@
+package org.wordpress.android.ui.reader.repository
+
+import kotlinx.coroutines.CoroutineDispatcher
+import org.wordpress.android.datasets.ReaderPostTable
+import org.wordpress.android.models.ReaderPostList
+import org.wordpress.android.models.ReaderTag
+import org.wordpress.android.modules.BG_THREAD
+import javax.inject.Inject
+import javax.inject.Named
+
+class FetchPostsForTagUseCase @Inject constructor(
```

Comment:
We usually use "fetch" for remote actions and "get" for local actions. We might want to rename this usecase to "GetPostsForTagUseCase". Wdyt?

Reply:
Good to know. I'll make the changes in the next commit. 👍